src/main/java/gov/usgs/wma/waterdata/groundwater/S3Bucket.java

```diff
@@ -22,7 +22,6 @@
 	protected String keyName;
 	protected File file;
 	protected Writer writer;
-	protected boolean isFileDisposable = true;
```

Comment:
This isn't used anywhere other than in testing, so I removed it.  Basically we always want to delete the tmp file.

Reply:
yep, I created that thinking I might need it. since we never have, to the chopping block with it.